wrappers/clang-target-wrapper.c

```diff
@@ -292,6 +295,23 @@ int _tmain(int argc, TCHAR* argv[]) {
     } else if (!_tcscmp(arch, _T("aarch64"))) {
         // Dwarf is the default here.
     }
+    
+    if (target_os && !_tcscmp(target_os, _T("mingw32uwp"))) {
+        // the UWP target is for Windows 10
+        exec_argv[arg++] = _T("-D_WIN32_WINNT=0x0A00 -DWINVER=0x0A00");
+        // the UWP target can only use Windows Store APIs
+        exec_argv[arg++] = _T("-DWINAPI_FAMILY=WINAPI_FAMILY_APP");
+        // the Windows Store API only supports Windows Unicode (some rare ANSI ones are available)
+        exec_argv[arg++] = _T("-DUNICODE");
+    } else if (target_os && !_tcscmp(target_os, _T("mingw32winrt"))) {
+        // the WinRT target is for Windows 8.1
+        exec_argv[arg++] = _T("-D_WIN32_WINNT=0x0603 -DWINVER=0x0603");
+        // the WinRT target can only use Windows Store APIs
+        exec_argv[arg++] = _T("-DWINAPI_FAMILY=WINAPI_FAMILY_APP");
+        // the Windows Store API only supports Windows Unicode (some rare ANSI ones are available)
+        // TODO should be replaced by -municode
```

Comment:
After the last discussion, I think we actually shouldn't use `-municode` here by default, as that affects the entry point as well. So perhaps leave out the TODO comment.

Reply:
OK, done.